CHANGELOG.md

```diff
@@ -1,3 +1,168 @@
+# 60.2.0
```

Comment:
you should provide a top blurb as to what this release includes. if there are new APIs you should show a small code snippet of how to use it.

Reply:
please see https://github.com/material-components/material-components-ios/blob/0eedc7a13a8d55af364bc89bd56514f13b231a25/contributing/releasing.md#new-features--top-level-description

Reply:
Done.